pkg/tcpip/stack/stack.go

```diff
@@ -46,6 +48,41 @@ const (
 	resolutionAttempts = 3
 )
 
+const (
+	// fakeNetNumber is used as a protocol number in tests.
+	// This constant should match fakeNetNumber in stack_test.go
+	fakeNetNumber    tcpip.NetworkProtocolNumber = math.MaxUint32
+)
+
+// Packet forwarding flags. Forwarding settings for different network protocols
+// are stored as bit flags in an uint32 number.
+const (
+	forwardingIPv4 uint32 = 1 << iota
```

Comment:
Consider adding a new type for this. It can just be an alias of uint32.

Reply:
added